pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools",
+    "Cython>=0.29.18",
+    "numpy==1.14.5; python_version=='3.6' and platform_system!='AIX'",
+    "numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'",
```

Comment:
These two should be numpy==1.15.1 as that is now our minimum supported version.

Reply:
I've replaced all the special cases with a single line, and used a `>=`. Is that going to bite?

Presumably there's a good reason scipy had it the way they did, but I don't have AIX around to test.